src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const basePrice = days * 40;
```

Comment:
Consider using a constant for the daily rental cost instead of hardcoding the value `40`. This will make the code more maintainable and understandable.